### PR TITLE
SFI-403 Authenticate with crown hosting

### PR DESCRIPTION
**Pull request by davidjamesstone**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -10,3 +10,5 @@ services:
     environment: 
       APPINSIGHTS_CLOUDROLE: ffc-sfi-crown-hosting-prototype
       APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY}
+      CH_API_ID: ${CH_API_ID},
+      CH_API_KEY : ${CH_API_KEY}
```
Comment (paulsimonandrews) [inline docker-compose.yaml]: Remove whitespace

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-sfi-crown-hosting-prototype",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (paulsimonandrews) [inline package.json]: Probably a minor release as you've added functionality, so `1.2.0`?

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
All looks good and can confirm it's working from the cluster.

Couple of tiny changes.

File: docker-compose.yaml (hunk 1 above)
Comment (davidjamesstone) [reply docker-compose.yaml]: Thanks, well spotted.

**Review by paulsimonandrews — APPROVED** [review]

**Pull request closed without merging**